source/application/AMPIC887DriverServer/AMPIC887ConsoleApplication.cpp

```diff
@@ -99,10 +579,80 @@ bool AMPIC887ConsoleApplication::startup()
 
 bool AMPIC887ConsoleApplication::startupControllers()
 {
-	controllerCollection_.append(new AMPIC887Controller("MC1611-401", "10.52.48.26"));
+	AMPIC887Controller* mc1611_401 = new AMPIC887Controller("MC1611-401", "10.52.48.26");
+	connect(mc1611_401, SIGNAL(output(QString)), this, SLOT(onControllerOutputReceived(QString)));
+	connect(mc1611_401, SIGNAL(errorEncountered(QString)), this, SLOT(onControllerErrorEncountered(QString)));
+
+	controllerCollection_.append(mc1611_401);
 
 	return (controllerCollection_.activeController() != 0);
 }
 
+void AMPIC887ConsoleApplication::makeConnections()
+{
+	connect(commandParser_, SIGNAL(help()), this, SLOT(onHelpCommandIssued()));
+	connect(commandParser_, SIGNAL(quit()), this, SLOT(onQuitCommandIssued()));
+	connect(commandParser_, SIGNAL(status()), this, SLOT(onStatusCommandIssued()));
+	connect(commandParser_, SIGNAL(changeActiveController(QString)), this, SLOT(onActiveControllerChangeIssued(QString)));
```

Comment:
To be consistent with every other signal/slot combo you made this could probably be `activeControllerChanged(QString)` and `onActiveControlChanged(QString)`.